src/haxeLanguageServer/Configuration.hx

```diff
@@ -81,6 +88,7 @@ private typedef UserConfig = {
 	var ?codeGeneration:CodeGenerationConfig;
 	var ?exclude:Array<String>;
 	var ?postfixCompletion:PostfixCompletionConfig;
+	var ?organizeImportsSortOrder:OrganizeImportsSortOrderConfig;
```

Comment:
I would just call this `importsSortOrder`. This is not necessarily specific to "Organize Imports", we might eventually want to insert auto-generated imports in the "correct place" as well.